src/language-js/utils/is-tsx-file.js

```diff
@@ -0,0 +1,3 @@
+export default function isTSXFile(options) {
```

Comment:
Optional minor suggestion: StrictCamelCase for the function name:

```suggestion
export default function isTsxFile(options) {
```